server/src/test/java/org/opensearch/cluster/ClusterStateTests.java

```diff
@@ -87,7 +87,7 @@ public void testSupersedes() {
         final DiscoveryNodes nodes = DiscoveryNodes.builder().add(node1).add(node2).build();
         ClusterName name = ClusterName.CLUSTER_NAME_SETTING.getDefault(Settings.EMPTY);
         ClusterState noMaster1 = ClusterState.builder(name).version(randomInt(5)).nodes(nodes).build();
```

Comment:
`noMaster1` -> `noClusterManager1`

Reply:
Ah, good catch! I missed it. 🏅 Thank you!

Reply:
Changed the variable in the new commit https://github.com/opensearch-project/OpenSearch/pull/2520/commits/6b756ec877fd1e395e4e3d2477ed6e5f04d29160